cmd/siac/rentercmd.go

```diff
@@ -503,9 +550,8 @@ func rentercontractsviewcmd(cid string) {
 	if err != nil {
 		die("Could not get contract details: ", err)
 	}
-	contracts := append(rc.Contracts, rc.OldContracts...)
 
-	for _, rc := range contracts {
+	for _, rc := range rc.Contracts {
```

Comment:
need to update this to pull all active, inactive, and expired contracts to then search through